### PR TITLE
Hotfix: sensor reading type route failing

### PR DESCRIPTION
**Pull request by seinnlee**
This PR fixes the issue where `sensor/:farm_id/reading_type` endpoint was not returning anything.

### CODE REVIEW

File: packages/api/src/routes/sensorRoute.js [hunk 1]
```diff
@@ -44,8 +44,8 @@ router.get('/:location_id/reading', SensorController.getAllReadingsByLocationId)
 router.get('/reading/farm/:farm_id', SensorController.getReadingsByFarmId);
 router.post('/reading/invalidate', SensorController.invalidateReadings);
 router.post('/unclaim', SensorController.retireSensor);
-router.get('/:location_id/reading_type', SensorController.getSensorReadingTypes);
 router.get('/:farm_id/reading_type', SensorController.getAllSensorReadingTypes);
+router.get('/:location_id/reading_type', SensorController.getSensorReadingTypes);
```
Comment (fsayyid) [inline packages/api/src/routes/sensorRoute.js:48]: The issue is that we can't have two endpoints with the format `:id/reading_type` express isn't able to tell the difference. Please update to `/farm/:farm_id/reading_type` on both frontend and backend.

**Review by fsayyid — CHANGES_REQUESTED** [review]

**Comment by Cali0707** [comment]
@seinnlee if I'm understanding this properly, this PR just changes which endpoint will fail, correct?

File: packages/api/src/routes/sensorRoute.js (hunk 1 above)
Comment (seinnlee) [reply packages/api/src/routes/sensorRoute.js:47]: That makes sense. I updated the endpoint both front and backend.

**Comment by seinnlee** [comment]
@Cali0707 Per @fsayyid's comment, I changed the endpoint so it's different from the other reading_type endpoint.

**Review by Cali0707 — APPROVED** [review]
LGTM!

**Review by fsayyid — APPROVED** [review]

**Pull request merged**